test/sqllogictest/range.slt

```diff
@@ -839,2343 +839,4367 @@ SELECT a::text t, array_agg(v::text ORDER BY v) FROM (
 [1,)
 {"(,1)","[-1,1)"}
 
-
 #
-# int8range
+# int4range union
 
 query T
-select '[0,100)'::int8range::text;
+SELECT (null + 'empty'::int4range)::text
 ----
-[0,100)
+NULL
 
 query T
-select pg_typeof('[0,100)'::int8range);
+SELECT ('empty'::int4range + 'empty'::int4range)::text
 ----
-int8range
+empty
 
 query T
-select 'empty'::int8range::text;
+SELECT ('empty'::int4range + 'empty'::int4range)::text
```

Comment:
I think the ``empty `op` empty`` cases are duplicated for each operator.

Reply:
Removed these and other duplicates that slipped in.